packages/website/ts/containers/zero_ex_js_documentation.ts

```diff
@@ -9,14 +9,18 @@ import { DocPackages } from 'ts/types';
 import { Translate } from 'ts/utils/translate';
 
 /* tslint:disable:no-var-requires */
-const IntroMarkdownV1 = require('md/docs/0xjs/1.0.0/introduction');
-const InstallationMarkdownV1 = require('md/docs/0xjs/1.0.0/installation');
-const AsyncMarkdownV1 = require('md/docs/0xjs/1.0.0/async');
-const ErrorsMarkdownV1 = require('md/docs/0xjs/1.0.0/errors');
-const versioningMarkdownV1 = require('md/docs/0xjs/1.0.0/versioning');
+const IntroMarkdownV1 = require('md/docs/0xjs/0.0.1/introduction');
+const InstallationMarkdownV1 = require('md/docs/0xjs/0.0.1/installation');
+const AsyncMarkdownV1 = require('md/docs/0xjs/0.0.1/async');
+const ErrorsMarkdownV1 = require('md/docs/0xjs/0.0.1/errors');
+const versioningMarkdownV1 = require('md/docs/0xjs/0.0.1/versioning');
+
+const IntroMarkdownV2 = require('md/docs/0xjs/1.0.0/introduction');
+const InstallationMarkdownV2 = require('md/docs/0xjs/1.0.0/installation');
+const AsyncMarkdownV2 = require('md/docs/0xjs/1.0.0/async');
+const ErrorsMarkdownV2 = ErrorsMarkdownV1;
+const versioningMarkdownV2 = require('md/docs/0xjs/1.0.0/versioning');
```

Comment:
This is slightly confusing. Perhaps it should be `IntroMarkdownV0` above and `IntroMarkdownV1` here? The versions here should align to the version that introduced the change. To be truly accurate, this should be `1.0.1`.